### PR TITLE
drop php 8.0, allow php 8.3

### PR DESCRIPTION
**Pull request by fezfez**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,15 +21,15 @@
...
+            "php": "8.1.99"
         },
         "allow-plugins": {
             "dealerdirect/phpcodesniffer-composer-installer": true
         }
     },
     "extra": {},
     "require": {
-        "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
+        "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
```
Comment (Ocramius) [inline composer.json:32]: `ErrorHandlerTest` requires a code removal here: otherwise OK

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (fezfez) [reply composer.json:32]: i just updated phpunit, the required changes are still needed ?

Comment (fezfez) [reply composer.json:32]: edit : ok good to go

**Comment by fezfez** [comment]
@Ocramius dco is clear and phpunit have been updated

**Review by Ocramius — APPROVED** [review]
Thanks @fezfez!

**Pull request merged**